ml/lin_reg.py

```diff
@@ -14,6 +14,10 @@ def __init__(self, learning_rate=1e-3, iters=10_000, batch_size=32) -> None:
         self.bias = None
 
     def fit(self, X: np.ndarray, y: np.ndarray) -> None:
+
+        if len(X[0]) <= 185:
```

Comment:
place this after line 21 and do `if num_samples <= 185`, then set `self.bias = 0` so that it's not `None`. i guess it could also just be initialized as zero in the constructor as well, instead of doing it here

Reply:
added that now sounds good